### PR TITLE
Update translation.ts

### PR DESCRIPTION
**Pull request by kenny136**
Translation added.

### CODE REVIEW

File: packages/ezcomments_extension/ezextension/ezcomments/translations/ger-DE/translation.ts [hunk 1]
```diff
@@ -246,11 +246,11 @@
     </message>
     <message>
         <source>reCAPTCHA API key missing.</source>
-        <translation type="unfinished"></translation>
+        <translation>reCAPTCHA API Schlüssel fehlt.</translation>
     </message>
     <message>
         <source>The words you input are incorrect.</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Die eingegebenen Wörter sind falsch.</translation>
```
Comment (lolautruche) [inline packages/ezcomments_extension/ezextension/ezcomments/translations/ger-DE/translation.ts:253]: Why do you leave it _unfinished_ ?

File: packages/ezcomments_extension/ezextension/ezcomments/translations/ger-DE/translation.ts [hunk 2]
```diff
@@ -325,11 +325,11 @@
     </message>
     <message>
         <source>Enter both words below, with or without a space.</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Geben Sie beide Wörter mit oder ohne Leerzeichen ein.</translation>
```
Comment (lolautruche) [inline packages/ezcomments_extension/ezextension/ezcomments/translations/ger-DE/translation.ts:328]: unfinished ?

File: packages/ezcomments_extension/ezextension/ezcomments/translations/ger-DE/translation.ts [hunk 3]
```diff
@@ -325,11 +325,11 @@
...
     </message>
     <message>
         <source>The letters are not case-sensitive.</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Bei den Buchstaben wird nicht zwischen Groß- und Kleinschreibung unterschieden.</translation>
```
Comment (lolautruche) [inline packages/ezcomments_extension/ezextension/ezcomments/translations/ger-DE/translation.ts:332]: unfinished ?

File: packages/ezcomments_extension/ezextension/ezcomments/translations/ger-DE/translation.ts [hunk 4]
```diff
@@ -337,7 +337,7 @@
     </message>
     <message>
         <source>Try another</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Versuche ein anderes</translation>
```
Comment (lolautruche) [inline packages/ezcomments_extension/ezextension/ezcomments/translations/ger-DE/translation.ts:340]: unfinished ?

Comment (kenny136) [inline packages/ezcomments_extension/ezextension/ezcomments/translations/ger-DE/translation.ts:340]: Depends on the context. So I'm not quite sure

File: packages/ezcomments_extension/ezextension/ezcomments/translations/ger-DE/translation.ts (hunk 3 above)
Comment (kenny136) [inline packages/ezcomments_extension/ezextension/ezcomments/translations/ger-DE/translation.ts:332]: Depends on the context. So I'm not quite sure

File: packages/ezcomments_extension/ezextension/ezcomments/translations/ger-DE/translation.ts (hunk 2 above)
Comment (kenny136) [inline packages/ezcomments_extension/ezextension/ezcomments/translations/ger-DE/translation.ts:328]: Depends on the context. So I'm not quite sure

File: packages/ezcomments_extension/ezextension/ezcomments/translations/ger-DE/translation.ts (hunk 1 above)
Comment (kenny136) [inline packages/ezcomments_extension/ezextension/ezcomments/translations/ger-DE/translation.ts:253]: Depends on the context. So I'm not quite sure

**Comment by lolautruche** [comment]
Thanks @kenny136 !

**Comment by SylvainGuittard** [comment]
Thanks @kenny136 and @lolautruche

**Comment by nfrp** [comment]
Many thanks guys.

**Pull request merged**